core-web/libs/dotcms-scss/angular/dotcms-theme/components/_confirmpopup.scss

```diff
@@ -75,10 +71,11 @@
     border-bottom-color: transparent;
 }
 
-.p-confirm-popup .p-confirm-popup-content {
+div.p-confirm-popup .p-confirm-popup-content {
```

Comment:
Why don't use another class of primeNG that this component uses instead of `div`? maybe `p-component` or something else